lib/interface.rb

```diff
@@ -0,0 +1,32 @@
+class Interface
+  attr_reader :order, :order_total
+
+  def initialize
+    @menu = Menu.new
+    @send = SendMessage.new
+    @order = {}
```

Comment:
It feels like order should be its own class.  "Interface" has two responsibilities: providing an interface for the user and dealing with orders.